CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## 2021.7.0 (7 July 2021)
+## 2021.8.9 (8 July 2021)
```

Comment:
The version in stable right now is `v2021.8.1007436305`. So we've gotta do this mess again. I'll fix the release plan to avoid this happening again.

Reply:
Should be 2021.8.0